### PR TITLE
[CI] Add devutils

### PR DESCRIPTION
**Pull request by eliyahu77**
Installs required services (kafka, redis, etc) for featurestore system tests

### CODE REVIEW

File: automation/system_test/dev_utilities.py [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+def status(redis, kafka, mysql, redisinsight):
+    namespace = "devtools"
+    if redis:
+        svc_password = get_svc_password(namespace, "redis", "redis-password")
+        print_svc_info(
+            "redis-master-0.redis-headless.devtools.svc.cluster.local",
+            6379,
+            "default",
+            svc_password,
+            "-------",
```
Comment (Tankilevitch) [inline automation/system_test/dev_utilities.py:317]: printing password doesn't feel secure to me

File: automation/system_test/dev_utilities.py [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+@click.option("--redis", is_flag=True, help="Install Redis")
+@click.option("--kafka", is_flag=True, help="Install Kafka")
+@click.option("--mysql", is_flag=True, help="Install MySQL")
+@click.option("--redisinsight", is_flag=True, help="Install Redis GUI")
+def status(redis, kafka, mysql, redisinsight):
+    namespace = "devtools"
+    if redis:
+        svc_password = get_svc_password(namespace, "redis", "redis-password")
+        print_svc_info(
+            "redis-master-0.redis-headless.devtools.svc.cluster.local",
```
Comment (Tankilevitch) [inline automation/system_test/dev_utilities.py:313]: what is the `redis-master-0` and `redis-headless`

File: automation/system_test/dev_utilities.py [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+            "password": svc_password,
+        }
+        return dict
+    if svc == "kafka":
+        dict = {"app_url": "kafka-0.kafka-headless.devtools.svc.cluster.local:9092"}
+        return dict
+    if svc == "mysql":
+        svc_password = get_svc_password(namespace, "mysql", "mysql-root-password")
+        dict = {
+            "app_url": "mysql-0.mysql.devtools.svc.cluster.local:3306",
```
Comment (Tankilevitch) [inline automation/system_test/dev_utilities.py:346]: same

File: automation/system_test/dev_utilities.py (hunk 2 above)
Comment (eliyahu77) [reply automation/system_test/dev_utilities.py:313]: this is the svc endpoint

File: automation/system_test/dev_utilities.py (hunk 1 above)
Comment (eliyahu77) [reply automation/system_test/dev_utilities.py:317]: it only when u call function status, it for next steps

**Pull request merged**